pkg/controller/perconaservermongodb/users.go

```diff
@@ -134,7 +140,7 @@ func (r *ReconcilePerconaServerMongoDB) reconcileUsers(ctx context.Context, cr *
 		}
 	}
 
-	internalSysSecretObj.Data = sysUsersSecretObj.Data
+	internalSysSecretObj.Data = getInternalSecretData(&sysUsersSecretObj)
```

Comment:
Do you need `1.19.0` check here as well?

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1731/commits/0a65f401d16165008dff9c32da0adf7ec9a4ff67